client/src/lib/components/auction/auction-admin.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import { useLands } from '$lib/api/land.svelte';
+  import type { Token } from '$lib/interfaces';
+  import BuySellForm from '../buy/buy-sell-form.svelte';
+  import { Button } from '../ui/button';
+  import { Input } from '../ui/input';
+  import { Label } from '../ui/label';
+  import {
+    Select,
+    SelectContent,
+    SelectItem,
+    SelectTrigger,
+    SelectValue,
+  } from '../ui/select';
+  import data from '$lib/data.json';
+  import { goto } from '$app/navigation';
+
+  let landStore = useLands();
+
+  let location = $state<number>(0);
+  let selectedToken = $state<Token | null>(null);
+  let stakeAmount = $state<number>(100);
+  let sellAmount = $state<number>(10);
+  let decayRate = $state<number>(2);
+
+  const handleCreateAuction = () => {
+    console.log('Creating Auction');
+    //TODO change the params
+    landStore
+      ?.auctionLand(
+        location,
+        stakeAmount,
+        sellAmount,
+        selectedToken?.address ?? '',
```

Comment:
```suggestion
        selectedToken?.address!,
```
We want to fail fast, not show the "Impossible to deserialize" screen